### PR TITLE
add some new devices

### PR DESCRIPTION
**Pull request by linoul**

### CODE REVIEW

File: devices/jdwhale.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
+const utils = require('zigbee-herdsman-converters/lib/utils');
+const tz = require('zigbee-herdsman-converters/converters/toZigbee');
+const exposes = require('zigbee-herdsman-converters/lib/exposes');
+const e = exposes.presets;
+const fz = require('zigbee-herdsman-converters/converters/fromZigbee');
+const reporting = require('zigbee-herdsman-converters/lib/reporting');
+
+module.exports = [
```
Comment (Koenkk) [inline devices/jdwhale.js:8]: Are these the following devices? If yes the vendor looks to be `J.XUAN`? https://www.jd.id/product/j-xuan-prz01-zigbee-intelligent-human-body-sensor-mobile-sen_626005876/626005877.html

File: devices/roome.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+const fz = require('zigbee-herdsman-converters/converters/fromZigbee');
+const exposes = require('zigbee-herdsman-converters/lib/exposes');
+const e = exposes.presets;
+
+module.exports = [
+    {
+        zigbeeModel: ['MULTI-MECI--EA01'],
+        model: 'HSC1-WD-0',
```
Comment (Koenkk) [inline devices/roome.js:8]: Can you provide an URL of this device? (with picture)

**Comment by Koenkk** [comment]
Note to self: after this merge https://github.com/Koenkk/zigbee2mqtt.io/pull/730/

File: devices/roome.js (hunk 2 above)
Comment (linoul) [reply devices/roome.js:8]: Only find the review link
https://www.smarthome.com.cn/xinpin/796.html
The manufacturer may be
http://en.mirsz.cn/detail-55.shtml

File: devices/jdwhale.js (hunk 1 above)
Comment (linoul) [reply devices/jdwhale.js:8]: Yes, it may need to be renamed j.xuan

**Comment by Koenkk** [comment]
Thanks!

**Pull request merged**